app/(afterLogin)/prompt/query-option.ts

```
@@ -0,0 +1,69 @@
+
+export const promptMutationOptions = {
+  create: () => {
+    return mutationOptions({
+      mutationFn: async () => {
+        // API 호출 로직 구현
+        try {
+          // const response = await createPrompt();
+          // return response;
+        } catch (error) {
+          throw new Error("프롬프트 생성 실패");
+        }
```

Biome (2.4.4)

[error] This code is unreachable

(lint/correctness/noUnreachable)